Users/Tutorial.rst

```diff
@@ -351,7 +351,7 @@ To get help on using a bear or to get a description of the bear, use the
     coala --bears=SpaceConsistencyBear --show-bears --show-description
```

Comment:
first of all, for the commit shortlog: It is not explicit enough. a better oen would be: ``Tutorial: Fix typo "of"->"or"``

as for the body, it is unneeded here at all, as the shortlog is explicit enough :) the body is optional, for situations like this

and for the last line, please give the full issue URL, as #2781 is not going to change anything in the correct repo :D